generated/nidcpower/nidcpower.proto

```diff
@@ -670,6 +672,16 @@ message ClearInterchangeWarningsResponse {
   int32 status = 1;
 }
 
+message ClearLatchedOutputCutoffStateRequest {
+  nidevice_grpc.Session vi = 1;
+  string channel_name = 2;
+  sint32 output_cutoff_reason = 3;
```

Comment:
Drat, there's an enum that should go with this. I'll spin up another metadata PR.

Reply:
Resolved with d288